orquesta/tests/unit/conducting/test_workflow_conductor_with_items.py

```diff
@@ -142,6 +142,99 @@ def test_basic_items_list(self):
         expected_output = {'items': task_ctx['xs']}
         self.assertDictEqual(conductor.get_workflow_output(), expected_output)
 
+    def test_basic_items_list_with_different_types(self):
+        wf_def = """
+        version: 1.0
+
+        vars:
+          - xs:
+              - null
+              - ""
+              - foobar
+              - 0
+              - 1
+              - 123
+              - 123.456
+              - False
```

Comment:
this should probably be `false` and `true` because of JSON not using capital letters, right?

Reply:
This is YAML. It seems to convert to False and True fine.